src/main/java/org/codehaus/mojo/wagon/CopyMojo.java

```diff
@@ -60,6 +61,20 @@ public class CopyMojo
     @Parameter( property = "wagon.caseSensitive")
     private boolean caseSensitive = true;
 
+    /**
+     * Local directory to store downloaded artifacts.
+     */
+    @Parameter( property = "wagon.downloadDirectory")
+    private File downloadDirectory;
```

Comment:
This needs way more description. If not set, what is the default? WHEN do I use it?